### PR TITLE
Fix/create pages

### PR DESCRIPTION
**Pull request by Earlo**

### CODE REVIEW

**Comment by Earlo** [comment]
Something to note!

https://github.com/strapi/strapi/issues/10322

File: gatsby-node.js [hunk 1]
```diff
@@ -25,31 +25,32 @@ exports.createPages = async ({ actions, graphql }) => {
...
-  data.allStrapiPage.nodes.forEach((page) => {
+  const buildPathTree = (key, loc) => {
+    const node = data.allStrapiPage.nodes.find((obj) => obj.slug === key)
+    if (!node.strapiParent) {
+      return loc !== "fi" ? `${loc}/${node.slug}` : node.slug
+    }
+    return `${buildPathTree(node.strapiParent.slug, loc)}/${node.slug}`
+  }
+  data.allStrapiPage.nodes.forEach((node) => {
+    const p = buildPathTree(node.slug, node.locale)
```
Comment (lakaniemi) [inline gatsby-node.js]: Use more descriptive variable name

File: gatsby-node.js [hunk 2]
```diff
@@ -25,31 +25,32 @@ exports.createPages = async ({ actions, graphql }) => {
...
+          title
           locale
-          content
         }
       }
     }
   `)
-
-  data.allStrapiPage.nodes.forEach((page) => {
+  const buildPathTree = (key, loc) => {
```
Comment (lakaniemi) [inline gatsby-node.js]: Maybe use more descriptive variable names. What is key/loc?

File: gatsby-node.js [hunk 3]
```diff
@@ -25,31 +25,32 @@ exports.createPages = async ({ actions, graphql }) => {
...
+    const node = data.allStrapiPage.nodes.find((obj) => obj.slug === key)
+    if (!node.strapiParent) {
+      return loc !== "fi" ? `${loc}/${node.slug}` : node.slug
+    }
+    return `${buildPathTree(node.strapiParent.slug, loc)}/${node.slug}`
+  }
```
Comment (lakaniemi) [inline gatsby-node.js]: Maybe a small description about this function's logic could be useful as it's not super self-explanatory

**Review by lakaniemi — APPROVED** [review]

**Pull request merged**